pyls/python_ls.py

```diff
@@ -234,7 +236,7 @@ def m_text_document__did_close(self, textDocument=None, **_kwargs):
     def m_text_document__did_open(self, textDocument=None, **_kwargs):
         self.workspace.put_document(textDocument['uri'], textDocument['text'], version=textDocument.get('version'))
         self._hook('pyls_document_did_open', textDocument['uri'])
-        self.lint(textDocument['uri'])
+        self.lint(textDocument['uri'], is_saved=True)
```

Comment:
I'm not sure this is true, isn't this the case when a new document is opened in the editor?

Reply:
Good point. Probably safer to assume it isn't saved (I could also check to see if the file exists on disk in `did_open`, but the way I read the spec it sounds like `did_open` just means that the language client is now handling the document, which doesn't necessarily mean it is in a saved state).

Reply:
(I went ahead and removed the logging regardless; none of the other linters do any)